README.md

```diff
@@ -0,0 +1,153 @@
+python-inspector - inspect Python packages dependencies and metadata
+=====================================================================
+
+Copyright (c) nexB Inc. and others.
+SPDX-License-Identifier: Apache-2.0
+Homepage: https://github.com/aboutcode-org/python-inspector and https://www.aboutcode.org/
+
+``python-inspector`` is a collection of utilities to:
+
+- resolve PyPI packages dependencies
+
+- parse various requirements.txt files and setup.py files as input
+  for resolving dependencies.
+
+- parse additionally various manifests and packages files such as
+  Pipfile, pyproject.toml, poetry.lock and setup.cfg and legacy and
+  current metadata file formats for eggs, wheels and sdist. These
+  have not been wired with the command line yet.
+
+- query PyPI JSON and simple APIs for package information
+
+It grew out of ScanCode toolkit to find and analyze PyPI archives and
+installed Python packages and their files.
+
+The goal of python-inspector is to be a comprehensive library
+that can handle every style of Python package layouts, manifests and lockfiles.
+
+Developing
+----------
+
+- [Install Astral Uv](https://docs.astral.sh/uv/getting-started/installation/). For convenience:
+
+  - Isolated: `pipx install uv`
+  - Regular: `pip install uv`
+
+- (Optional) Configure pre-commit for commit linter checks:
+
+```bash
+   pre-commit install
+   pre-commit install --hook-type commit-msg
+```
+
+- Run from development. A virtual .venv will be created if you not have one
+
+```bash
+uv run python-inspector --help
+```
+
+Testing
+--------
+
+- Run the tests. Tests have a special dependency group with their requirements for text exclusively:
+
+```bash
+uv sync --group=test
+uv pytest -vvs
+```
+
+- These are live tests to regenrate the tests with updated data run::
+
+```bash
+uv sync
+PYINSP_REGEN_TEST_FIXTURES=yes uv run pytest -vvs
+```
+
+Documentation
+-------------
+
+```bash
+uv sync --all-groups
+hatch run validate-docs
```

Comment:
Why introduce yet another tool with hatch? This should be its own PR to understand and discuss the benefits.